ports/biomassgrowth/BIOMASS Growth.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+# device_info.txt will be included by default
+
+export PORT_32BIT="Y"
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+GAMEDIR="/$directory/ports/biomassgrowth"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+# Log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+# Extract the game files
+if [ -f "$GAMEDIR/gamedata/Biomass Growth 20211213.exe" ]; then
+    
+	cd "$GAMEDIR" || exit
+		
+	# Use 7zip to extract the Biomass Growth 20211213.exe file to the destination directory
+    "$GAMEDIR/patch/7zzs" -aos x "$GAMEDIR/gamedata/Biomass Growth 20211213.exe" -o"$GAMEDIR/gamedata" & pid=$!
+
+    # Wait for the extraction process to complete
+    wait $pid
+
+    # Check if the Biomass Growth 20211213.exe file exists
+    if [ -f "$GAMEDIR/gamedata/Biomass Growth 20211213.exe" ]; then
```

Comment:
It was already tested in the `if` statement line 34. Do we need to test again ? I haven't looked at the content of `Biomass Growth 20211213.exe` so maybe I am missing something :)